bin/webpack.js

```diff
@@ -1,21 +1,71 @@
 #!/usr/bin/env node
+function runCommand(command, options) {
+	const cp = require("child_process");
+	return new Promise((resolve, reject) => {
+		const executedCommand = cp.spawn(command, options, {
+			stdio: "inherit"
+		});
+
+		executedCommand.on("error", error => {
+			reject(error);
+		});
+
+		executedCommand.on("exit", code => {
+			if (code === 0) {
+				resolve(true);
+			} else {
+				reject();
+			}
+		});
+	});
+}
 
 let webpackCliInstalled = false;
 try {
 	require.resolve("webpack-cli");
 	webpackCliInstalled = true;
-} catch (e) {
+} catch (err) {
 	webpackCliInstalled = false;
 }
 
-if (webpackCliInstalled) {
-	require("webpack-cli"); // eslint-disable-line node/no-missing-require, node/no-extraneous-require, node/no-unpublished-require
+if (!webpackCliInstalled) {
+	const path = require("path");
+	const fs = require("fs");
+	const inquirer = require("inquirer");
```

Comment:
Could you please use readLine package that is built in node? I don't think we need the whole inquirer library for this task..

Reply:
I do agree about that, but as others pointed out on the referenced issue, inquirer is "more beautiful" and provides a better user experience. What are your thoughts on that @sokra ?

Reply:
In this case I would prefer the lightweight package, as it's a one-time question and we could save npm size in this way.

Reply:
PS: Don't forget to `close()` the readline instance to allow webpack-cli to claim stdin/out

Reply:
Alright, later today i'm going to do that.